sharks/src/share_ff.rs

```diff
@@ -9,15 +9,15 @@ use zeroize::Zeroize;
 
 use crate::ff::*;
 
-pub const FIELD_ELEMENT_LEN: usize = 32;
+pub const FIELD_ELEMENT_LEN: usize = 24;
 
 #[cfg_attr(feature = "fuzzing", derive(Arbitrary))]
 #[cfg_attr(feature = "zeroize_memory", derive(Zeroize))]
 #[derive(PrimeField)]
-#[PrimeFieldModulus = "52435875175126190479447740508185965837690552500527637822603658699938581184513"]
-#[PrimeFieldGenerator = "7"]
+#[PrimeFieldModulus = "680564733841876926926749214863536421547"] // 2^129 - 1365
+#[PrimeFieldGenerator = "3"]
```

Comment:
How was this new generator chosen?

Reply:
The new generator was just chosen for simplicity, since it is smaller than 7 and we are operating in a prime-order group anyway.